framework/generated/khronos_generators/vulkan_generators/encode_pnext_struct_generator.py

```diff
@@ -183,33 +193,8 @@ def endFile(self):
         # Finish processing in superclass
         BaseGenerator.endFile(self)
 
-    def genStruct(self, typeinfo, typename, alias):
-        """Method override."""
-        if not alias:
-            # Only process struct types that specify a 'structextends' tag, which indicates the struct can be used in a pNext chain.
-            parent_structs = typeinfo.elem.get('structextends')
-            if parent_structs:
-                stype = self.make_structure_type_enum(typeinfo, typename)
-                if stype:
-                    self.stype_values[typename] = stype
-
     def need_feature_generation(self):
         """Indicates that the current feature has C++ code to generate."""
-        if self.stype_values:
+        if self.feature_extended_structs:
```

Comment:
Same comment re: "needed" above

Reply:
As mentioned above about we currently need it for `needs_feature_generation`.  Maybe we can remove in the future.